src/renderer/task-runner.ts

```diff
@@ -102,7 +105,11 @@ export class TaskRunner {
 
   private async setup(req: SetupRequest) {
     const { log } = this;
-    const { fiddle, hideChannels, showChannels, version } = req;
+    const { fiddle, hideChannels, showChannels, useObsolete, version } = req;
+
+    if (useObsolete === true || useObsolete === false) {
```

Comment:
Isn't this... always true?

Reply:
no. If the user doesn't specify it on the command line, its value is undefined.

I guess I could use `typeof useObsolete == 'boolean'` to be clearer?